components/director/internal/open_resource_discovery/validation_test.go

```diff
@@ -1364,13 +1364,14 @@ func TestDocuments_ValidateBundle(t *testing.T) {
 			},
 		},
 		{
-			Name: "Missing `shortDescription` field for Bundle",
+			Name: "Valid missing `shortDescription` field for Bundle",
```

Comment:
Add tests for descriptionLables similarly to labels for each entity.

Also, add a test about version rule is checking only the major component of the version.